src/parser/nodes.rs

```diff
@@ -119,7 +119,7 @@ pub struct CommandNode {
     /// [`TreeNode`]: struct.TreeNode.html
     pub node: TreeNode,
     /// The handler which is executed once this node has been accepted.
-    pub handler: Option<fn(&node: Node) -> ()>,
+    pub handler: Option<fn(node: Node) -> ()>,
```

Comment:
This should actually be `node: &Node` probably as I don't mean for the `handler` to consume the `node`. Right?